TwilioVerifySDK/TwilioVerify/Sources/Domain/Manager/TwilioVerify.swift

```diff
@@ -30,6 +30,21 @@ public typealias FactorListSuccessBlock = ([Factor]) -> ()
 ///:nodoc:
 public typealias EmptySuccessBlock = () -> ()
 
+/// Wrapper for [kSecAttrAccessible](https://developer.apple.com/documentation/security/ksecattraccessible)
+public enum KeyAttrAccessible {
+  case afterFirstUnlock
+  case afterFirstUnlockThisDeviceOnly
+
+  var value: CFString {
+    switch self {
+      case .afterFirstUnlock:
+        return kSecAttrAccessibleAfterFirstUnlock
+      case .afterFirstUnlockThisDeviceOnly:
+        return kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly
+    }
+  }
+}
+
 ///Describes the available operations to proccess Factors and Challenges
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/5d0f0dc7fcbbb876a57dfd6fbf71f6a800f5ce49/TwilioVerifySDK/TwilioVerify/Sources/Domain/Manager/TwilioVerify.swift#L48"></span>Prefer at least one space after slashes for comments.<br />
<code>comment_spacing</code> <code>TwilioVerify.swift:48</code></td>
    </tr>
  </tbody>
</table>